### ISSUE TITLE
Error creating Meteor app

### ISSUE THREAD

**Issue by rvanhoepen** [meteor/windows-preview#12]
I'm getting an error when trying to create a meteor app. It throws out this error message: "Error creating Meteor App. Unable to update package catalog (are you offline?)"

I'm behind a proxy, but I was able to install Meteor. Tried running Powershell as administrator to see if that might be the problem, also tried in WebStorm where my proxy is definitely set up correctly, but I get the same output.

**Comment by raghumanbabu** [comment]
Hi,
I installed the windows preview 0.1.1 version. I can able to create a meteor app. But, I am trying to add some packages like "meteor add urigo:angular" but, I am getting an issue which is pasted below:

Unable to update package catalog (are you offline?)
 => Errors while adding packages:
While downloading mquandalle:bower@0.1.11...:
error: getaddrinfo ENOTFOUND
While downloading urigo:angular@0.6.1...:
error: getaddrinfo ENOTFOUND
Your package catalog may be out of date.
Please connect to the internet and try again.

Is it because of proxy? How to solve this issue?

**Comment by mattkrick** [comment]
sub! still a problem in 0.1.7
Error below:
```
Unable to update package catalog (are you offline?)
 => Errors while adding packages:
While downloading momentjs:moment@2.8.4...:
error: UNABLE_TO_VERIFY_LEAF_SIGNATURE
Your package catalog may be out of date.
Please connect to the internet and try again.
```

**Comment by smeijer** [comment]
```
D:\dev\myApp>meteor update --release WINDOWS-PREVIEW@0.1.7
=> Errors while initializing project:

While building package velocity:test-proxy:
error: File not found: tests/mocha/server/myTest.js
error: File not found: tests/mocha/server/myTest1.js
error: File not found: tests/mocha/server/myTest2.js
...
```

**Comment by Slava** [comment]
Hey guys, so I have just tried to run behind a proxy on my VM and it worked for me as I set the environment variables for Meteor. Here is what I've done:

```batch
SET HTTP_PROXY=http://user:password@1.2.3.4:5678
SET HTTPS_PROXY=http://user:password@1.2.3.4:5678
meteor update
```

Note the `http://` protocol part for both HTTP and HTTPS (your settings might differ).

The `UNABLE_TO_VERIFY_LEAF_SIGNATURE` is something that has to do with extra SSL certs by looks at it?

**Comment by jeffschmitz** [comment]
I'm getting something similar both when behind proxy with environment variables set, and when not behind a proxy with no environment variables.  Here's the error I'm seeing when starting meteor:

While selecting package versions:
error: Package version not in catalog: npm-container 1.0.0

While refreshing package catalog to resolve previous errors:
error: Network error: wss: //packages.meteor.com/websocket: getaddrinfo ENOTFOUND

**Comment by avital** [comment]
Looks like the comments on this issue are due to separate issues. Per @Slava's comment above, we have implemented proxy support in Meteor 1.1 for Windows, so I'll close this ticket.

If you have an issue, please open a new ticket on the [main Meteor repository](https://github.com/meteor/meteor/). This repository will be closed soon.
